.github/workflows/test-pr.yml

```diff
@@ -20,17 +20,15 @@ jobs:
           sudo apt-get update
           sudo apt-get install libharfbuzz-dev libfreetype-dev
       if: matrix.os == 'ubuntu-latest'
+    - name: Install dependencies for MacOS
+      run: |
+          brew install pkg-config freetype harfbuzz
+      if: matrix.os == 'macos-latest'
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: python -m pip install cython numpy
-    - name: Install library (w/ text rendering support)
+    - name: Install library and dependencies
       run: python -m pip install -e .
-      if: matrix.os == 'ubuntu-latest'
-    - name: Install library (w/o text rendering support)
-      run: python -m pip install -e . --install-option --no-text-rendering
-      if: matrix.os != 'ubuntu-latest'
```

Comment:
Can we keep the explicit no-text-rendering part? Using the `CELIAGG_NO_TEXT_RENDERING` flag in the environment?

Reply:
To be more clear: It was decided in the early days of this project that one should be able to build the library with only numpy if they don't care about missing text rendering. So as long as that functionality is in place, we should endeavor to test it.

This decision can of course be revisited, but that's not the intent of this PR.

Reply:
My concern was that we had a branch of the text rendering code (the windows non-freetype rendering) that is being missed from tests - ie. if something breaks text rendering on windows, we wouldn't know.

It seems like a richer matrix might make sense, perhaps at the expense of testing less Python versions on Windows/Mac.

What I might do is revert this to testing exactly the same set of cases as the original, and there can be another PR around what cases we actually want to test.

Reply:
Sounds good. That's a very good point about explicitly testing Windows text rendering.